histoDB/histodb-dataserver/pom.xml

```diff
@@ -153,11 +153,16 @@
         </dependency>
 
         <dependency>
+            <groupId>eu.itesla_project</groupId>
+            <artifactId>ucte-import</artifactId>
+            <version>${ipstentsoe.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
I was pretty sure you added the ucte-import dependency for this reason. This dependency (and probably cim1-import) is in the compile scope. Maybe, we can simply change the scope (runtime ?) to include them in the uber jar, but to keep the histodb-dataserver fully independant of the importers. What do you think?